Blues/Peripheral/Peripheral.swift

```diff
@@ -78,6 +78,10 @@ open class Peripheral: NSObject, PeripheralProtocol {
         self.core = core
         self.queue = queue
     }
+    
+    open func updateAdvertisement(_ advertisement: Advertisement) {
```

Comment:
Is this intended to be empty? If yes, could you maybe add a comment with an explanation?